addons/mail/static/src/core/public_web/user_status_panel.scss

```diff
@@ -0,0 +1,21 @@
+.o_card_avatar_im_status {
+    bottom: -4px;
+    right: -4px;
+    border-radius: 2rem;
+    background-color: white;
+    height: 18px;
+    width: 18px;
+}
+
+o-min-width-0 {
+    min-width: 0;
+}
```

Comment:
`min-w-0` does that (yes there's still our `o-min-width-0` that we should scrap)